DipUI/DipUI/StoryboardInstantiatable.swift

```diff
@@ -0,0 +1,80 @@
+//
+// DipUI
+//
+// Copyright (c) 2016 Ilya Puchka <ilyapuchka@gmail.com>
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+#if os(iOS) || os(tvOS)
+  import UIKit
+  typealias Storyboard = UIStoryboard
+  typealias Responder = UIResponder
+#else
+  import AppKit
+  typealias Storyboard = NSStoryboard
+  typealias Responder = NSResponder
+#endif
+
+import Dip
+
+public protocol StoryboardInstantiatable {
+  func instantiatedFromStoryboard(withTag tag: DependencyContainer.Tag, container: DependencyContainer)
```

Comment:
Seems the semantic and naming conventions are generally more leaning to a name like `didInstantiate(fromStoryboard storyboard, withTag tag, container:)`